src/AuthHelper.php

```
@@ -115,12 +188,56 @@ public static function getInstance(?string $secret = null, string $algorithm = '
+    /**
+     * Try to validate OIDC token from Zitadel
+     *
+     * Uses OidcClient for JWKS handling and token validation.
+     * Prefers ID token for user profile information (preferred_username, email, name, etc.)
+     * as the access token typically only contains minimal claims (sub).
+     *
+     * @param string $issuer Zitadel issuer URL
+     * @param string $clientId Zitadel client ID
+     * @return object|null Validated payload or null
+     */
+    private static function tryValidateOidcToken(string $issuer, string $clientId): ?object
```

PHPMD (2.15.0)

Avoid unused parameters such as '$issuer'. (undefined)

(UnusedFormalParameter)